src/main/resources/templates/demoAdmin.html

```diff
@@ -241,13 +241,13 @@
                                         <span ng-if="options.mqType === 'KAFKA'" translate="KAFKA_CONFIGURATION_DETAILS"></span>
                                         <span ng-if="options.mqType === 'RABBIT_MQ'" translate="RABBIT_MQ_CONFIGURATION_DETAILS"></span>
                                     </a>
-                                    <span ng-if="options.mqType === 'KAFKA' && displayKafkaError === 'true'">
-                                        ( <i class="fa fa-exclamation-circle error_icon" aria-hidden="true"></i>
-                                        <span translate="INVALID_KAFKA_SERVER_URL"></span> )
+                                    <span class="invalid_message" ng-if="options.mqType === 'KAFKA' && displayKafkaError === 'true'">
+                                        <i class="fa fa-exclamation-circle" aria-hidden="true"></i>
+                                        <span translate="INVALID_KAFKA_SERVER_URL"></span>
```

Comment:
For Kafka:
![image](https://user-images.githubusercontent.com/109124579/211993367-93dec364-f51a-406f-a5fe-b93288525cde.png)

For RabbitMQ:
![image](https://user-images.githubusercontent.com/109124579/211993337-ab231419-e728-46e0-a843-c0417f2f876f.png)